components/footer/styled-components.ts

```diff
@@ -9,6 +9,10 @@ export const PageContainer = styled('div')(({ theme }) => ({
   [theme.breakpoints.up('lg')]: {
     padding: '120px 0 154px',
   },
+
+  '&.footer-secondary': {
```

Comment:
As far as I see, we don't need these styles. I believe we can make unification and use the same padding on all the pages.

Reply:
'+